inspire_dojson/hep/rules/bd9xx.py

```diff
@@ -272,6 +273,10 @@ def references2marc(self, key, value):
 
     journal_title = get_value(reference, 'publication_info.journal_title')
     journal_volume = get_value(reference, 'publication_info.journal_volume')
+    if journal_title and journal_volume:
+        if journal_title[-1] in string.ascii_uppercase:
```

Comment:
This is wrong, it would catch `JHEP`.

Reply:
True, and we're missing a test for that.

Reply:
Is checking for ending in `\.[A-Z]` enough, then?